2023/ego-and-math/spanish/sentence_translations.json

```
@@ -622,7 +622,7 @@
  },
  {
   "input": "I did a series about linear algebra I did a series about calculus people asked if I could explain what is a Fourier transform or how do neural networks work and so I obliged. ",
-  "translatedText": "Hice una serie sobre álgebra lineal. Hice una serie sobre cálculo. La gente me preguntó si podía explicar qué es una transformada de Fourier o cómo funcionan las redes neuronales y acepté. ",
+  "translatedText": "Hice una serie sobre álgebra lineal. Hice una serie sobre cálculo. La gente me preguntó si podía explicar qué es una transformacion de Fourier o cómo funcionan las redes neuronales y acepté. ",
```

```suggestion
  "translatedText": "Hice una serie sobre álgebra lineal. Hice una serie sobre cálculo. La gente me preguntó si podía explicar qué es una transformada de Fourier o cómo funcionan las redes neuronales y acepté. ",
```
The correct term in Spanish is actually `transformada` even if it doesn't seem like it: https://es.wikipedia.org/wiki/Transformada_de_Fourier